src/utils/displayUtils.test.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
You're a saint for adding tests. Thank you!

Reply:
You're welcome. I felt like I should add more, and will if I have some time on my hands, but I can also do that in a separate PR later on, so don't let that hold up merging this unless you feel something important is missing.